kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/dhcp/server/DnsmasqTool.java

```diff
@@ -121,14 +130,21 @@ private boolean isConfigFileAlteredOrNonExistent(String interfaceName)
         }
     }
 
-    private Command systemctlRestartCommand() {
-        return new Command(new String[] { "systemctl", "restart", DhcpServerTool.DNSMASQ.getValue() });
+    private boolean removeInterfaceConfig(String interfaceName) {
+        try {
+            File configFile = new File(DhcpServerManager.getConfigFilename(interfaceName));
+            return Files.deleteIfExists(configFile.toPath());
+        } catch (Exception e) {
+            return false;
+        }
     }
 
     private void writeGlobalConfig() {
         try {
             Path dnsmasqGlobalsPath = Paths.get(DNSMASQ_GLOBAL_CONFIG_FILE);
-            Files.write(dnsmasqGlobalsPath, GLOBAL_CONFIGURATION.getBytes(StandardCharsets.UTF_8));
+            if (Files.notExists(dnsmasqGlobalsPath)) {
```

Comment:
What happens if someone changes the content of the file outside Kura? Is it possible to check the hash of this file as well?